mono_repo/lib/src/commands/pub.dart

```diff
@@ -54,17 +53,13 @@ Future<Null> pub(String pubCommand,
   print(lightBlue
       .wrap('Running `pub $pubCommand` across ${configs.length} package(s).'));
 
-  var processManager = new ProcessManager();
-
   for (var dir in configs.keys) {
     print('');
     print(wrapWith('Starting `$dir`...', [styleBold, lightBlue]));
     var workingDir = p.join(rootDirectory, dir);
 
-    // TODO(kevmoo): https://github.com/dart-lang/io/issues/22
-    Directory.current = workingDir;
-
-    var proc = await processManager.spawn('pub', [pubCommand]);
+    var proc = await Process.start('pub', [pubCommand],
```

Comment:
nit: might as well fix this to do `p.join(p.dirname(Platform.resolvedExecutable), 'pub')`

Reply:
Ah! Good idea!